src/components/groomers/GroomerDisplay.js

```diff
@@ -1,22 +1,18 @@
 import React, { useState, useEffect } from 'react';
 import { getGroomerInfo } from '../../api/index';
 import axios from 'axios';
-
 const GroomerDisplay = props => {
   const [groomers, setGroomers] = useState([]);
   const groomId = window.location.pathname.split('/groomers/').pop();
-
   // const getGroomerProfile = (id) => {
   //   return axios
   //     .get(`${process.env.REACT_APP_API_URI}/groomers/${id}`)
   //     .then(response => response.data)
   //     .catch(err => console.log(err));
   // };
-
   // useEffect(() => {
   //   getGroomerInfo(groomId).then(response => {
   //     setGroomers(response);
-
   //   });
   // }, []);
```

Comment:
Not a major thing right now, but let's try to keep commented out code out of our PR's into main